script/script.js

```diff
@@ -415,6 +512,61 @@ const treetable = {
         }
     }
 };
+const saveCategory = function () {
+    categoriesCollection.add({
+        value: "NEW CATEGORY"
```

Comment:
just FYI, why we add a new category with prefilled  value

Reply:
I can agree that an empty category is better. 
moreover, you can open the cell with name for editing: 

~~~
var id = table.add({});
table.edit({row:id, column:"name");
~~~

https://docs.webix.com/api__link__ui.datatable_edit.html

Reply:
done